.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 2.6.2
```

Comment:
Could you remove this from the PR? (or add `.tool-versions` to the `.gitignore` -- either is fine!)

Reply:
of course, i'm sorry about that.

Reply:
Not a problem! It happens :)